build/buildRunTimeImageBases.sh

```diff
@@ -123,24 +124,45 @@ for dockerFile in $dockerFiles; do
     platformName="${PARTS[0]}"
     platformVersion="${PARTS[1]}"
 
-    # Set $localImageTagName to the following format: oryxdevmcr.azurecr.io/public/oryx/base:{platformName}-{platformVersion}
-    localImageTagName="$BASE_IMAGES_REPO:$platformName-$platformVersion"
+    if shouldStageRuntimeVersion $platformName $platformVersion ; then
```

Comment:
I originally had code that would rewrite base.buster.Dockerfile based on whether the version was to be staged or not, but honestly there were enough changes like comments and things that this ended up being a more straightforward way to implement. In the future I think we could try using templates or something to swap values of a single file, but with the time crunch for this, I didn't think the added complexity was worth it.